// should return error

fun() {
return 3
return 4
}
